### PR TITLE
Address issue 1107

### PR DESCRIPTION
**Pull request by jmvera255**
minor text modification to provide further clarification of $2, $3 special variables when they are first introduced in the episode re: issue #1107

### CODE REVIEW

File: _episodes/06-script.md [hunk 1]
```diff
@@ -147,10 +147,16 @@ ATOM     13  H           1      -1.183   0.500  -1.412  1.00  0.00
...
 
-We still need to edit `middle.sh` each time we want to adjust the range of lines,
-though.
-Let's fix that by using the special variables `$2` and `$3` for the
-number of lines to be passed to `head` and `tail` respectively:
+Currently, we need to edit `middle.sh` each time we want to adjust the range of 
+lines that is returned. 
+Let's fix that by configuring our script to instead use three command-line arguments. 
+After the first command-line argument ($1), each additional argument that we 
+provide will be assigned a special variable name of increasing interger values
```
Comment (gcapes) [inline _episodes/06-script.md:154]: typo: integer

File: _episodes/06-script.md [hunk 2]
```diff
@@ -147,10 +147,16 @@ ATOM     13  H           1      -1.183   0.500  -1.412  1.00  0.00
...
-We still need to edit `middle.sh` each time we want to adjust the range of lines,
-though.
-Let's fix that by using the special variables `$2` and `$3` for the
-number of lines to be passed to `head` and `tail` respectively:
+Currently, we need to edit `middle.sh` each time we want to adjust the range of 
+lines that is returned. 
+Let's fix that by configuring our script to instead use three command-line arguments. 
+After the first command-line argument ($1), each additional argument that we 
+provide will be assigned a special variable name of increasing interger values 
+such that $1, $2, $3, etc. refer to the first, second, third, etc. command-line
```
Comment (gcapes) [inline _episodes/06-script.md:155]: These two mid-sentence et ceteras don't read well. I think we can safely remove them both, please.

File: _episodes/06-script.md (hunk 1 above)
Comment (jmvera255) [reply _episodes/06-script.md]: I have corrected the typo in the third commit

File: _episodes/06-script.md (hunk 2 above)
Comment (jmvera255) [reply _episodes/06-script.md]: I have removed 'etc.' in the third commit

File: _episodes/06-script.md [hunk 3]
```diff
@@ -147,10 +147,16 @@ ATOM     13  H           1      -1.183   0.500  -1.412  1.00  0.00
...
 
-We still need to edit `middle.sh` each time we want to adjust the range of lines,
-though.
-Let's fix that by using the special variables `$2` and `$3` for the
-number of lines to be passed to `head` and `tail` respectively:
+Currently, we need to edit `middle.sh` each time we want to adjust the range of 
+lines that is returned. 
+Let's fix that by configuring our script to instead use three command-line arguments. 
+After the first command-line argument (`$1`), each additional argument that we 
+provide will be assigned a special variable name of increasing values
```
Comment (gcapes) [inline _episodes/06-script.md:154]: ... will be accessible via the special variables `$1`, `$2`, `$3` etc

Comment (jmvera255) [reply _episodes/06-script.md]: I like this suggested change and have updated my PR accordingly

**Comment by gcapes** [comment]
Great work - a nice improvement to the material. Thanks.

**Pull request merged**